src/bin/add/args.rs

```diff
@@ -184,7 +184,11 @@ impl Args {
             }
 
             if let Some(ref path) = self.path {
-                dependency = dependency.set_path(dunce::canonicalize(path)?);
+                let dep_path = dunce::canonicalize(path)?;
+                let manifest = get_manifest_from_path(&dep_path)?;
+
+                dependency = dependency.set_available_features(manifest.features());
+                dependency = dependency.set_path(dep_path);
```

Comment:
In `crate_name.rs` we don't pass the canonicalized path to `get_manifest_from_path`.  I only bring this up because I had a lot of issues making it work correctly, I'm a bit paranoid about how it all works together :)

Reply:
fair, I'll use the uncanonicalized version then. thanks :)